mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -195,7 +184,7 @@ def explode_action(
         mrp_date_demand = mrp_action_date
         if mrp_date_demand < date.today():
             mrp_date_demand = date.today()
-        bom = self._get_bom_to_explode(product_mrp_area_id)
+        bom = product_mrp_area_id.supply_bom_id
```

Comment:
For keeping extensibility, can you keep the method `_get_bom_to_explode` and return `product_mrp_area_id.supply_bom_id` in it?